.circleci/config.yml

```diff
@@ -16,27 +20,16 @@ jobs:
           command: ./coursier bootstrap org.scalameta:scalafmt-cli_2.12:2.0.0-RC4 -r bintray:scalameta/maven -o scalafmt --main org.scalafmt.cli.Cli
 
       - run:
-          name: install zip
-          command: apt-get update && apt-get install -y --no-install-recommends zip
-
-      - run:
-          name: install bazel
-          command: ~/repo/docker/installBazel
+          name: install mill
+          command: sh -c '(echo "#!/usr/bin/env sh" && curl -L https://github.com/lihaoyi/mill/releases/download/0.4.1/0.4.1) > /usr/local/bin/mill && chmod +x /usr/local/bin/mill'
 
       - run:
           name: test
-          command: |
-             bazel test --jobs=1 //... --test_output=errors
-
-      - run:
-          name: build
-          command: |
-            bazel build --jobs=1 //src/io/iohk/scalanet:scalanet_EXT_deploy.jar
-
-            cp bazel-bin/src/io/iohk/scalanet/scalanet_EXT_deploy.jar scalanet.jar
-
-            .circleci/removeScalaStandardLibrary scalanet.jar
+          command: mill scalanet.test && mill scalanet.scoverage.htmlReport
```

Comment:
scoverage reports 89.69% coverage, looks good :-)

Reply:
Yes, Shailesh has written quite a few. We'll be playing the test pack story, so hopefully it will go upto 100% or very close.